### PR TITLE
test: enhance tests

### PR DESCRIPTION
**Pull request by mattsse**

### CODE REVIEW

**Review by yash-atreya — APPROVED** [review]

**Review by grandizzy — APPROVED** [review]
🙏

File: crates/test-utils/src/util.rs [hunk 1]
```diff
@@ -893,6 +893,15 @@ impl TestCommand {
         self.assert().success()
     }
 
+    /// Runs the command and asserts that it resulted in success, with expected JSON data.
+    #[track_caller]
+    pub fn assert_json_response(&mut self, expected: Inline) {
```
Comment (DaniPopes) [inline crates/test-utils/src/util.rs:898]:
```suggestion
    pub fn assert_json_response(&mut self, expected: impl IntoData) {
```

Comment (DaniPopes) [inline crates/test-utils/src/util.rs:898]: `assert_json_stdout`?

Comment (grandizzy) [reply crates/test-utils/src/util.rs]: yep, changed

**Pull request merged**